data/json/terrain.json

```diff
@@ -2886,6 +2899,7 @@
         "type" : "terrain",
         "id" : "t_window_bars_alarm",
         "name": "window with metal bars",
+        "description": "A giant sheet of glass inserted into a window with thick security grilles, making it impossible to crawl through.  Typically installed for high-value stores, or at least stores in bad neighborhoods.  This one has a small sticker in a corner stating, 'Protected by AtmoWeb, leading AI in terminating crime'.",
```

Comment:
> Protected by AtmoWeb, leading AI in terminating crime

I like what you've done here, but this is definitely misleading, right? Doesn't this just ring an alarm when breached? "Terminating crime" seems a bit extreme

Reply:
This is just meant to give a tiny hint to anyone that takes the time to read the description that this one is alarmed (even in real life, sometimes the best deterrent is a security sticker). Secondly, it's meant to be an easter egg/reference to Terminator series.

Atmo = Sky, Web = Net, leading _AI_ in _terminating_ crime

And I mean, the broken window triggers Eyebots, which then summon security bots (which can be lethal if you're not able to handle them) if you're spotted.